gov2/aurora/actions/clusters.go

```diff
@@ -0,0 +1,356 @@
+package actions
+
+import (
+	"context"
+	"errors"
+	"log"
+
```

Comment:
Why the blank line here?

Reply:
That's a Go standard format thing. Built-in imports are separated from external imports by a blank line.